docs/dev_guide/newcomers.rst

```diff
@@ -1 +1,269 @@
-.. include:: ../../CONTRIBUTING.rst
+.. _newcomers:
```

Comment:
Copied this here for now.

Reply:
Isn't this somewhat defeating the purpose? It's fine for now though.

Reply:
Sure it is but I don't have a better idea.

Reply:
yeah I guess the only real alternative is to use MyST and download it on build.

Reply:
I will do that.

Reply:
Which I am not sure I am on board with. That being said maybe we should re-write `CONTRIBUTING.md` to be more generic and higher level and link to this?

Reply:
yeah maybe, I wonder if anyone else has opinions.

Reply:
Will make the project wide one the same as the website.